File: java/src/main/java/za/co/entelect/challenge/entities/Lane.java
package za.co.entelect.challenge.entities;

import com.google.gson.annotations.SerializedName;
import za.co.entelect.challenge.enums.Terrain;

public class Lane {
    @SerializedName("position")
    public Position position;

    @SerializedName("surfaceObject")
    public Terrain terrain;

    @SerializedName("occupiedByPlayerId")
    public int occupiedByPlayerId;
}

File: dotnetcore/StarterBot/Entities/Lane.cs
using Newtonsoft.Json;
using StarterBot.Enums;

namespace StarterBot.Entities
{
    public class Lane
    {
        public MapPosition Position { get; set; }
        public CellType SurfaceObject { get; set; }
        public PlayerIdentifier PlayerIdentifier { get; set; }
    }
}